# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/nvme-cli/hotfix-basename.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- nvme-cli-2.11/plugins/netapp/netapp-nvme.c.vanilla	2024-12-14 14:55:21.623290080 +0100
+++ nvme-cli-2.11/plugins/netapp/netapp-nvme.c	2024-12-14 15:22:38.516615391 +0100
@@ -22,6 +22,7 @@
 #include <unistd.h>
 #include <errno.h>
 #include <string.h>
+#include <libgen.h>
 
 #include "common.h"
 #include "nvme.h"
